app/src/org/commcare/activities/BiometricsHelper.java

```diff
@@ -0,0 +1,81 @@
+package org.commcare.activities;
```

Comment:
Maybe better to move it out of `org.commcare.utils`

Reply:
No problem.